### DISCUSSION TITLE
[PhoneAuthProvider] How to set the default country code

### DISCUSSION THREAD

**Discussion by jasonlaw** [firebase/flutterfire#10442] [Q&A]
How do we set the default country for the selection list?
Besides, if it is possible to show only a selected list of country code?

**Comment by rusudinu** [comment]
You can set the default country code like this:

```
PhoneNumberField(
  controller: myController,
  defaultSelectedCountryIsoCode: 'US', // Set default country to United States
)
```

if you want to show only a couple of countries you can do it, like this:

```
PhoneNumberField(
  controller: myController,
  enabledCountries: ['US', 'CA', 'MX'], // Show only United States, Canada, and Mexico
)
```

**Reply by jasonlaw** [reply]
Thanks for the reply.
I can't find the PhoneNumberField widget.
I am using firebase_ui_auth: ^1.1.9.

**Reply by rusudinu** [reply]
it's this one: https://pub.dev/packages/phonenumbers/example

**Reply by rusudinu** [reply]
you could also use this one: https://pub.dev/packages/intl_phone_number_input
